packages/jest-runner/fuzz.ts

```diff
@@ -63,6 +63,18 @@ export function fuzz(
 		// affecting other tests, e.g. set a test specific timeout.
 		const localConfig = JSON.parse(JSON.stringify(fuzzingConfig));
 
+		if (process.env.JAZZER_LIST_FUZZTEST_NAMES != undefined) {
+			if (process.env.JAZZER_LIST_FUZZTEST_NAMES_PATTERN != undefined) {
+				if (
+					name.toString().match(process.env.JAZZER_LIST_FUZZTEST_NAMES_PATTERN)
+				) {
+					console.log(name);
+				}
+			} else {
+				console.log(name);
```

Comment:
AFAIK `cifuzz` only checks `stderr`. Could you please clarify that?

Reply:
As discussed in https://codeintelligence.slack.com/archives/C04T229T87P/p1695313766046999 stdout is fine, so I'll keep it! :)